tests/e2e/env/docker-compose.yml

```diff
@@ -38,6 +38,6 @@ services:
     container_name: wcp_e2e_phpmyadmin
     image: phpmyadmin/phpmyadmin:latest
     ports:
-      - "8093:80"
+      - "8085:80"
```

Comment:
Any reason for the E2E phpmyadmin port change?

Reply:
To keep it "next" to the port used by the e2e WP container (8084) as it is in our other setups and to avoid conflicts with people going outside the range, like it happened to me when I discovered 8093 was not free for some reason.